.github/workflows/ci.yml

```diff
@@ -63,9 +57,6 @@ jobs:
         run: npx playwright install chromium --with-deps
       - name: Wait for docker
         run: sleep 60
-      - name: Run e2e tests
```

Comment:
why are we removing the e2e tests in ci?

Reply:
We removed this originally to publish WIP. Added back.